AlphaGo/training/supervised_policy_trainer.py

```diff
@@ -6,276 +6,247 @@
 from keras import backend as K
 from AlphaGo.util import confirm
 from keras.optimizers import SGD
-from keras.callbacks import Callback
+from keras.callbacks import ModelCheckpoint, Callback
 from AlphaGo.models.policy import CNNPolicy
 from AlphaGo.preprocessing.preprocessing import Preprocess
 
 # default settings
-DEFAULT_MAX_VALIDATION = 1000000000
+DEFAULT_MAX_VALIDATION = 1e9
 DEFAULT_TRAIN_VAL_TEST = [.95, .05, .0]
-DEFAULT_LEARNING_RATE = .003
+DEFAULT_LEARNING_RATE = 3e-3
 DEFAULT_BATCH_SIZE = 16
-DEFAULT_DECAY = .0000000125
+DEFAULT_DECAY = 1.25e-8
 DEFAULT_EPOCH = 10
 
-# metdata file
-FILE_METADATA = 'metadata_policy_supervised.json'
-# weight folder
-FOLDER_WEIGHT = os.path.join('policy_supervised_weights')
+FILE_METADATA = 'metadata.json'
```

Comment:
We've attempted to set a precedent in the 'data' repository where results go into `training_results/supervised/<model_dataset>/`